crates/anthropic/src/anthropic.rs

```diff
@@ -89,52 +89,52 @@ impl Model {
         Self::Claude3_5Haiku
     }
 
-    pub fn from_id(id: &str) -> Result<Self> {
+    pub fn from_id(id: &str) -> Option<Self> {
```

Comment:
The only place we call this immediately does `.ok()` on the `Result` it returns, so I just had it return `Option` directly.

Reply:
I'm somewhat inclined to leave the `Result` given that this is "library code". It doesn't seem to be hurting anything to have it there, and returning just an `Option` results in a loss of information for callers that may be interested in it.

Reply:
Fair enough! 👍 

Reverted in 7970f234a971623a512258d203ee893f8abf45e5